### DISCUSSION TITLE
버튼이 눌리지 않습니다.

### DISCUSSION THREAD

**Discussion by choramha** [naverz/zepeto-studio-kor#786] [5. Q&A]
안녕하세요. 상호작용 버튼을 만들고 싶어서
https://github.com/naverz/zepeto-studio-kor/discussions/442#discussioncomment-2263512
 일단  링크에 나와있는대로 따라해봤는데, 버튼이 눌리지 않습니다. 
 캔버스 속성을 overlay로 바꾸면 문제없이 잘 눌리는데, world space로 설정하면 클릭이 안되네요. 
 어떻게 해결해야할까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 댓글 내용처럼 Event Camera를 제페토 카메라로 등록해야 클릭이 가능합니다.
스크립트를 추가해 주시고 
공식 샘플 월드 Chapter3에 상호작용 예제 파일이 있으니 참고하여 주세요.
감사합니다.
https://github.com/naverz/zepeto-studio-kor/discussions/615

**Reply by choramha** [reply]
카메라 등록이 안되어있었네요. 감사합니다!!!

**Comment by yoonjoohwan** [comment]
혹시 반대쪽으로 돌려서 클릭해보셨을까요...?
버튼 뒷면을 클릭하고 계실수도 있을것 같은데
